packages/cli/src/utils/path-util.ts

```diff
@@ -0,0 +1,16 @@
+import * as path from 'node:path';
+
+/**
+ * Checks if the given childPath is contained within the parentPath. Resolves
+ * the paths before comparing them, so that relative paths are also supported.
+ */
+export function isContainedWithin(parentPath: string, childPath: string): boolean {
```

Comment:
We should probably look into other places that need this at some point

Reply:
Definitely. There's at least on in `Server.ts` that I'll update after this is merged